tests/SprykerEcoTest/Zed/Easycredit/EasycreditOrderIdentifierSaverTest.php

```diff
@@ -19,14 +19,16 @@
 class EasycreditOrderIdentifierSaverTest extends AbstractEasycreditTest
 {
     /**
+     * @skip
```

Comment:
Why it's marked as skipped?

Reply:
Forgot to remove. Now fixed the test and removed `@skip`